src/lib/types.ts

```diff
@@ -33,9 +33,13 @@ export interface User {
   }
   aud: string
   confirmation_sent_at?: string
+  recovery_sent_at?: string
+  action_link?: string
   email?: string
+  phone?: string
   created_at: string
-  confirmed_at?: string
+  email_confirmed_at?: string
```

Comment:
we should add back `confirmed_at` and make it deprecated 

https://stackoverflow.com/questions/60755711/is-it-possible-to-mark-something-as-deprecated-in-typescript

Reply:
as discussed with @kiwicopple , on the assumption that some people might be using the `confirmed_at` column directly we can add it back in as a column in the auth schema as a generated column - which takes the minimum date of `email_confirmed_at`, and `phone_confirmed_at`, I'll start looking into it now (https://github.com/supabase/gotrue/issues/152)

https://www.codeproject.com/Articles/5246207/Generated-Columns-in-PostgreSQL